code/game/machinery/autolathe_datums.dm

```diff
@@ -399,6 +407,10 @@ datum/autolathe/recipe/medilathe/autoinjector/s30x6
 	name = "rollerbed"
 	path = /obj/item/roller
 
+/datum/autolathe/recipe/medilathe/surgerybed
```

Comment:
everything else seems fine, but i'm a bit concerned with the balance of adding even more surgical beds to the game. imo we should actually be reducing the amount

Reply:
Fairs, I'll remove them